mobile/lib/src/presentation/navigation/navdrawer_widget.dart

```diff
@@ -0,0 +1,81 @@
+import 'dart:io';
+
+import 'package:auto_route/auto_route.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:polydodo/src/application/navdrawer/navdrawer_bloc.dart';
+import 'package:polydodo/src/presentation/navigation/routes/router.gr.dart';
+
+class NavDrawerPage extends StatelessWidget {
+  static const name = "appDrawerRoute";
+
+  @override
+  Widget build(BuildContext context) {
+    return Drawer(
+      child: ListView(
+        padding: EdgeInsets.zero, //only(top: 8.0),
+        children: <Widget>[
+          _createHeader(),
+          _createDrawerItem(
+            icon: Icons.bluetooth,
+            text: "Bluetooth selector",
+            onTap: () {
```

Comment:
Create a function like _changeRoute(NavdrawerState state, Routes route) as arguments so code won't need to be changed for each drawerItem.

Reply:
Not sure how can each drawer item has the same changeRoute call :O ?

Reply:
`onTap: _changeRoute(NavdrawerState.BluetoothSelector, Routes.bluetoothSelectorPage),`

Reply:
And you'll need to define this:
```
void _changeRoute(BuildContext context, NavdrawerState nextState, Routes nextRoute) {
          context
              .bloc<NavdrawerBloc>()
              .add(NavdrawerUpdated(NavdrawerState.BluetoothSelector));
          ExtendedNavigator.of(context).replace(Routes.bluetoothSelectorPage);
}
```

Reply:
Hum you seem to be refering to an outdated version of the file